components/Quote/Quote.module.css

```diff
@@ -0,0 +1,28 @@
+.largecontainer,
+.container {
+	margin: 1em 0;
+	padding: 1em;
+    border-left-style: outset;
+    color: gray;
+}
+
+.largecontainer p {
+    font-size: 3rem;
+}
+
+.container cite,
```

Comment:
As we start writing more styles, we should consider switching the SCSS (or something else?)